lib/web_console/templates/console.js.erb

```diff
@@ -112,13 +113,18 @@ REPLConsole.prototype.install = function(container) {
   });
 
   // Initialize
-  this.inner = container.getElementsByClassName('console-inner')[0];
-  this.clipboard = document.getElementById('clipboard');
+  this.inner = this.findChild('console-inner');
+  this.clipboard = this.findChild('clipboard');
   this.remotePath = container.dataset.remotePath;
   this.newPromptBox();
   this.insertCss();
 };
 
+// Find an element by class name
+REPLConsole.prototype.findChild = function(className) {
```

Comment:
Does it make sense to inline this function, like you did with `getConsoleHeight`? We can "promote" it to prototype function (instance method) once we have other functions needing it.